test/e2e/shoot/create_migrate_and_delete.go

```diff
@@ -0,0 +1,113 @@
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package shoot
+
+import (
+	"context"
+	"strings"
+	"time"
+
+	"github.com/gardener/gardener/pkg/apis/core/v1beta1"
+	. "github.com/gardener/gardener/test/framework"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+const (
+	defaultMrExcludeList              = "extension-controlplane-shoot-webhooks,extension-shoot-dns-service-shoot,extension-worker-mcm-shoot"
+	defaultResourcesWithGeneratedName = "apiserver-proxy-config"
+)
+
+var _ = Describe("Shoot Tests", Label("CPM"), func() {
+	f := defaultShootCreationFramework()
+	f.Shoot = defaultShoot("migrate-")
+
+	It("Create, Migrate and Delete", func() {
+		By("Create Shoot")
+		ctx, cancel := context.WithTimeout(parentCtx, 15*time.Minute)
+		defer cancel()
+		Expect(f.CreateShootAndWaitForCreation(ctx, false)).To(Succeed())
+		f.Verify()
+
+		By("Migrate Shoot")
+		ctx, cancel = context.WithTimeout(parentCtx, 15*time.Minute)
+		defer cancel()
+		t, err := newDefaultShootMigrationTest(ctx, f.Shoot, f.GardenerFramework)
+		Expect(err).ToNot(HaveOccurred())
+		Expect(beforeMigration(ctx, t)).To(Succeed())
+		Expect(t.MigrateShoot(ctx)).To(Succeed())
+		Expect(afterMigration(ctx, t)).To(Succeed())
+
+		By("Delete Shoot")
+		ctx, cancel = context.WithTimeout(ctx, 15*time.Minute)
+		defer cancel()
+		Expect(f.DeleteShootAndWaitForDeletion(ctx, f.Shoot)).To(Succeed())
+	})
+})
+
+func newDefaultShootMigrationTest(ctx context.Context, shoot *v1beta1.Shoot, gardenerFramework *GardenerFramework) (*ShootMigrationTest, error) {
+	t, err := NewShootMigrationTest(ctx, gardenerFramework, &ShootMigrationConfig{
+		ShootName:               shoot.Name,
+		ShootNamespace:          shoot.Namespace,
+		TargetSeedName:          "local2",
+		SkipNodeCheck:           true,
+		SkipMachinesCheck:       true,
+		SkipProtectedToleration: true,
+	})
+	return t, err
+}
+
+func beforeMigration(ctx context.Context, t *ShootMigrationTest) error {
+	if t.Shoot.Status.IsHibernated {
+		return nil
+	}
+
+	By("Creating test Secret and Service Account")
+	return t.CreateSecretAndServiceAccount(ctx)
+}
+
+func afterMigration(ctx context.Context, t *ShootMigrationTest) error {
+	if CurrentSpecReport().Failed() {
+		t.GardenerFramework.DumpState(ctx)
+		return cleanUp(ctx, t)
+	}
+
+	By("Verifying migration...")
+	if err := t.VerifyMigration(ctx); err != nil {
+		return err
+	}
+
+	if t.Shoot.Status.IsHibernated {
+		return nil
+	}
+
+	By("Checking if the test Secret and Service Account are migrated ...")
+	if err := t.CheckSecretAndServiceAccount(ctx); err != nil {
```

Comment:
As @timebertt mentioned above, this cannot be executed currently when using skaffold (running inside prow) because it requires an entry in /etc/hosts. The call here even needs to be executed against the second seed, for which the istio-ingressgateway is exposed on port `9443`

Reply:
So we skip all that checks for now?